packages/frontend/web-ui/src/game/helpers/setFormFieldValue.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  NUMBER_PLAYERS_MAXIMUM,
+  NUMBER_PLAYERS_MINIMUM,
+} from './numberOfPlayersValues';
+
+export function setFormFieldValue(
```

Comment:
I'd say this function has no purpose with the new approach

Reply:
I'll delete it